src/components/SpotPicker.js

```diff
@@ -5,6 +5,17 @@ import useLocation from "../hooks/useLocation";
 
 import SelectWithDefault from "./SelectWithDefault";
 
+const sortByName = (items) => {
+  const sortedNames = items
+    .map(item => item.name)
+    .sort((a, b) => a.localeCompare(b));
```

Comment:
See [String.localeCompare on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)

If it was only A-Z we wouldn't need to use localeCompare, but it handles things like "São Tomé and Príncipe" in Africa. It takes as a second argument a locale to use, but I think it may be doing a smart lookup to use the right locale or something.

Reply:
I was just looking at that. Fancy.